public/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head><meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<title>Band App</title>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">  
+<head>
+
+<body>
+	<div class="container">
+
+		<!--nav-->
+		<nav class="navbar navbar-default">
+		  <div class="container-fluid">
+		    <div class="navbar-header">
+		      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+		        <span class="sr-only">Toggle navigation</span>
+		        <span class="icon-bar"></span>
+		        <span class="icon-bar"></span>
+		        <span class="icon-bar"></span>
+		      </button>
+		      <a class="navbar-brand" href="#">Home</a>
+		    </div>
+		    <!-- Collect the nav links, forms, and other content for toggling -->
+		    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+		      <ul class="nav navbar-nav">
+		        <li><a href="/bands">All Bands<span class="sr-only">(current)</span></a></li>
+		        <li><a href="/genres">All Genres</a></li>
+
+		      </ul>
+		    </div><!-- /.navbar-collapse -->
+		  </div><!-- /.container-fluid -->
+		</nav>
+ 		<!--page title-->
+ 		<div class="page-header">
+   		<h1>Discover your next fave band</h1>
+   		<p>Choose from genre dropdown below to find your next fave band via genre!</p>
+		</div><!-- /.page-header -->
+
+		<div class="dropdown">
+		  <button class="btn btn-default dropdown-toggle" type="button" id="genre" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+		    Genre
+		    <span class="caret"></span>
+		  </button>
+
+		  <ul class="dropdown-menu" aria-labelledby="dropdownMenu1">
+		    <li><a href="/genre/Chill%20Wave">Chill Wave</a></li>
+		    <li><a href="/genre/Dream%20Pop">Dream Pop</a></li>
+ 		    <li><a href="/genre/Electronic">Electronic</a></li>
+		    <li><a href="/genre/Trip%20Hop">Trip Hop</a></li>
+		  </ul>
+		</div>
+
+		<div id="band-output"; style="background-color:#c6c6c6; height:200px; overflow:hidden;">
+		   	<p>results of query go here</p>
+		   	  <ul class="filter-bands-by-genre-list"></ul>
+
+		</div>
+	</div><!--end container -->
+
+<script src="client.js"></script>
```

Comment:
You'll probably want your client.js to come after your jQuery script tag, especially if you're using jQuery in it.